playbooks/device-create.yaml

```diff
@@ -77,7 +77,7 @@
   stc: 
     action: config
     count: 2
-    objects: /EmulatedDevice[@Name='dev-$item']
+    objects: /EmulatedDevice[@Name='client-$item']
```

Comment:
Why is this needed?

Reply:
> Why is this needed?

device name is "client-$item" or "server-$item".

Reply:
ok